spec/operators/buffer-spec.ts

```diff
@@ -16,11 +17,25 @@ describe('Observable.prototype.buffer', () => {
     testScheduler.run(({ hot, expectObservable }) => {
       const a = hot('   -a-b-c-d-e-f-g-h-i-|');
       const b = hot('   -----B-----B-----B-|');
-      const expected = '-----x-----y-----z-|';
+      const expected = '-----x-----y-----z-(F|)';
```

Comment:
What you're going to see through a lot of these tests is I had to add the last emission of an empty buffer for cases when the `closingNotifier` was not complete when the source completed.

Reply:
Yeah, but TBH I am a little disappointed that you didn't use an emoji:
```ts
      const a = hot('   -a-b-c-d-e-f-g-h-i-|');
      const b = hot('   -----B-----B-----B-|');
      const expected = '-----x-----y-----z-(🦖|)';
      const expectedValues = {
        x: ['a', 'b', 'c'],
        y: ['d', 'e', 'f'],
        z: ['g', 'h', 'i'],
        '🦖': [],
      };
```